UpSkill_App/UpSkill.Infrastructure/Data/Models/Invoice.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+
+    public class Invoice<T1, T2>
```

Comment:
Hm, not sure we need such a complicated structure.
An invoice should be a collection of the courses + another collection of the coach sessions particular owner has paid for his/her employees for a given month/period.

So now that I think about it we might not event need an invoice as model as all of the necessary information could be extracted from other classes and just a ViewModel for displaying the info could be enough.

Reply:
I was just following the Wiki file, the contents of which we discussed on Saturday.